### DISCUSSION TITLE
Contributors API results inconsistent with contributors insights page

### DISCUSSION THREAD

**Discussion by xvello** [community/community#46421] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello folks!

Some of my project's users contribute new rules via issues instead of PRs, and I'm OK doing the commit myself. For them to still appear in the repo's contributors list, I list them as `Co-authored-by`, for example [in this commit](https://github.com/letsblockit/letsblockit/commit/e6599584981f90351e7b3b6e0511e3650449ac8a).

While these users are shown as contributors in the project's homepage and [insights page](https://github.com/letsblockit/letsblockit/graphs/contributors), they are not listed in the results of [the repo contributors API](https://api.github.com/repos/letsblockit/letsblockit/contributors), which I was planning to call, in order to show contributors on the project's website. My understanding is that the dataset backing the API only looks for commits' committers, and ignores the `Co-authored-by`, even if I set `anon=1`.

- Could you add a request param to also return these contributors, by querying the same data set as the insights page / repo homepage sidebar?
- In the meantime, what would be the best way to retrieve the full list?

**Comment by ArtyomVancyan** [comment]
I got the same issue when getting the contributors list through the GitHub API.

### Does not work as expected

https://github.com/pysnippet/fuzzymap (two contributors)
https://api.github.com/repos/pysnippet/fuzzymap/contributors (only one in the list)

https://github.com/pysnippet/pysnippet (two contributors)
https://api.github.com/repos/pysnippet/pysnippet/contributors (only one in the list)

### Works as expected

https://github.com/pysnippet/django-mermaid (two contributors)
https://api.github.com/repos/pysnippet/django-mermaid/contributors (lists both contributors)
